api/v1/Client/Start.cs

```diff
@@ -81,8 +81,7 @@ public static async Task hello(ClientOptions options)
 
         // Create a gRPC channel
         using var channel = GrpcChannel.ForAddress($"http://{options.RemoteAddr}");
-        var client = new MyGreeter.MyGreeterClient(channel);
-
+        var client = Client.NewClient(channel);
```

Comment:
Cant leave the comment at line 132, but consider using a common logging library across the service instead of using Console.WriteLine() in some scenarios

Reply:
Added client logging in most recent commit